api/envoy/extensions/expr/custom_cel_vocabulary/extended_request/v3/config.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.extensions.expr.custom_cel_vocabulary.extended_request.v3;
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.expr.custom_cel_vocabulary.extended_request.v3";
+option java_outer_classname = "ConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/expr/custom_cel_vocabulary/extended_request/v3;extended_requestv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Extended Request CEL Vocabulary Config]
+
+// This is an reference implementation of the extension category envoy.expr.custom_cel_vocabulary.
+// There exists a default envoy native CEL vocabulary for evaluating CEL expressions.
+// This config is used to indicate to the CEL expression evaluator that a custom CEL vocabulary
+// should also be used to evaluate CEL expressions.
+// Any fields which pertain to the custom CEL vocabulary can be added,
```

Comment:
In the input yaml, there's this block: 
```
        http_filters:          
          - name: rbac-deny-rule
            typed_config:
              "@type": type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC
              rules:
                action: DENY
                custom_cel_vocabulary_config:
                  name: envoy.expr.custom_cel_vocabulary.extended_request
                  typed_config:
                    "@type": type.googleapis.com/envoy.extensions.expr.custom_cel_vocabulary.extended_request.v3.ExtendedRequestCelVocabularyConfig
                    return_url_query_string_as_map: true
```

Here, `ExtendedRequestCelVocabularyConfig` has this configuration: `return_url_query_string_as_map`.  

But you could define a `ExtendedRequestCelVocabularyConfig` which is empty.  

It would indicate to the RBAC filter that a custom cel vocabulary is in use and to create an implementation instance and then to call `fillActivation` and `registerFunctions`.

Reply:
As per the other comment on documenting syntax, I think if we specify the full set of additional syntax elements here it would probably be clearest.

Reply:
You want a list of all the attributes and functions being added?

Reply:
Yep